nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ValidateCsv.java

```diff
@@ -446,6 +461,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
         final String schema = context.getProperty(SCHEMA).evaluateAttributeExpressions(flowFile).getValue();
         final CellProcessor[] cellProcs = this.parseSchema(schema);
         final boolean isWholeFFValidation = context.getProperty(VALIDATION_STRATEGY).getValue().equals(VALIDATE_WHOLE_FLOWFILE.getValue());
+        final boolean getAllViolations = context.getProperty(GET_ALL_VIOLATIONS).asBoolean();
```

Comment:
See earlier comment.
```suggestion
        final boolean allViolations = context.getProperty(ALL_VIOLATIONS).asBoolean();
```